src/app/game/card/shape/shape.component.html

```diff
@@ -1,25 +1,24 @@
+
 <!-- shape.component.html -->
 
-<div class="shape" [style.background-color]="shape.color">
-    <div [ngSwitch]="shape.type">
-      <div *ngSwitchCase="cardShapeType.Circle" class="circle"></div>
-      <div *ngSwitchCase="cardShapeType.Triangle" class="triangle"></div>
-      <div *ngSwitchCase="cardShapeType.Square" class="square"></div>
-      <div *ngSwitchCase="cardShapeType.Star" class="star"></div>
-      <div *ngSwitchCase="cardShapeType.Hexagon" class="hexagon"></div>
-      <div *ngSwitchCase="cardShapeType.Pentagon" class="pentagon"></div>
-      <div *ngSwitchCase="cardShapeType.Octagon" class="octagon"></div>
-      <div *ngSwitchCase="cardShapeType.Heart" class="heart"></div>
-      <div *ngSwitchCase="cardShapeType.Rhombus" class="rhombus"></div>
-      <div *ngSwitchCase="cardShapeType.Parallelogram" class="parallelogram"></div>
-      <div *ngSwitchCase="cardShapeType.Arrow" class="arrow"></div>
-      <div *ngSwitchCase="cardShapeType.Crescent" class="crescent"></div>
-      <div *ngSwitchCase="cardShapeType.Trapezoid" class="trapezoid"></div>
-      <div *ngSwitchCase="cardShapeType.Oval" class="oval"></div>
-      <div *ngSwitchCase="cardShapeType.Cross" class="cross"></div>
-      <div *ngSwitchCase="cardShapeType.Ring" class="ring"></div>
-      <!-- Add cases for more shapes as needed -->
-      <div *ngSwitchDefault class="default"></div>
-    </div>
+<div class="shape">
+  <div [ngSwitch]="shape.type">
+    <div *ngSwitchCase="cardShapeType.Rectangle" class="rectangle" [style.background-color]="shape.color"></div>
+    <div *ngSwitchCase="cardShapeType.Circle" class="circle" [style.background-color]="shape.color"></div>
+    <div *ngSwitchCase="cardShapeType.Arrow" class="arrow" [style.color]="shape.color"></div>
```

Comment:
ng class directive for dynamic class naming, for scalability if we change the shape name in the future...